discrete_summary_checker/example/Check_Format.ipynb

```diff
@@ -1180,7 +451,7 @@
    "name": "python",
    "nbconvert_exporter": "python",
    "pygments_lexer": "ipython3",
-   "version": "3.9.12"
+   "version": "3.9.16"
   }
  },
  "nbformat": 4,
```

Comment:
Code looks good. I downloaded a summary spreadsheet from Alfresco and this notebook seems to work as intended.